inst/default_options.csv

```diff
@@ -1,37 +1,37 @@
-ZWE,ZWE,2,CY2020Q4,CY2021Q3,CY2019Q3,ZWE2020PHIA,ZWE2020PHIA,"",false,true,true,true,true,2020,12,2017,2020,2017,2020,subnational,true,250,1000,1000,true,-4,true,1,true,1,true,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,true,false,true,false,false
-NGA,NGA,2,CY2020Q4,CY2021Q3,CY2018Q4,NGA2017AKIS,NGA2018NAIS,"",false,true,true,true,true,2019,12,2018,2019,2018,2019,subnational,true,250,1000,1000,false,-4,true,1,true,1,true,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,true,false,true,false,false
-SWZ,SWZ,1,CY2020Q4,CY2021Q3,CY2016Q4,SWZ2016PHIA,SWZ2016PHIA,"",false,true,true,true,true,2020,12,2016,2020,2016,2020,national,true,250,1000,1000,false,-4,true,1,true,1,true,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,true,false,true,false,false
-CIV,CIV,2,CY2020Q4,CY2021Q3,CY2017Q4,CIV2017PHIA,CIV2017PHIA,"",false,true,true,true,true,2020,12,2017,2020,2017,2020,national,true,250,1000,1000,false,-4,true,1,true,1,true,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,true,false,true,false,false
-XXX,XXX,2,CY2020Q4,CY2021Q3,CY2017Q4,CMR2018DHS; CMR2017PHIA,CIV2017PHIA,"",invalid,true,true,true,true,2020,12,2017,2020,2017,2020,national,true,250,1000,1000,false,-4,true,1,true,1,true,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,true,false,true,false,invalid
+iso3,area_scope,area_level,calendar_quarter_t2,calendar_quarter_t3,calendar_quarter_t1,survey_prevalence,survey_art_coverage,survey_recently_infected,use_survey_aggregate,include_art_t1,include_art_t2,artattend,artattend_t2,anc_clients_year2,anc_clients_year2_num_months,anc_prevalence_year1,anc_prevalence_year2,anc_art_coverage_year1,anc_art_coverage_year2,spectrum_population_calibration,output_aware_plhiv,max_iterations,no_of_samples,rng_seed,permissive,artattend_log_gamma_offset,use_kish_prev,deff_prev,use_kish_artcov,deff_artcov,use_kish_recent,deff_recent,spectrum_plhiv_calibration_level,spectrum_plhiv_calibration_strat,spectrum_artnum_calibration_level,spectrum_artnum_calibration_strat,spectrum_aware_calibration_level,spectrum_aware_calibration_strat,spectrum_infections_calibration_level,spectrum_infections_calibration_strat,calibrate_method,deff_vls,use_kish_vls,rho_paed_x_term,rho_paed_15to49f_ratio,alpha_xst_term,adjust_area_growth,anchor_home_district
+AGO,AGO,1,CY2020Q4,CY2021Q3,CY2015Q4,AGO2015DHS,,,FALSE,FALSE,FALSE,FALSE,FALSE,2020,12,2017,2020,2017,2020,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,none,sex_age_group,none,sex_age_coarse,none,sex_age_coarse,none,sex_age_coarse,logistic,1,TRUE,FALSE,TRUE,FALSE,FALSE,TRUE

```

Comment:
Is the change of format (from `true` -> `TRUE` and `""` -> ``) here from reading in R and then writing out again? I think empty cells will be fine but we should double check, the true TRUE change might cause issues, depends what the JSON serialization does

Reply:
No I edited in excel and saved back asa csv. What would you suggest as the preferred method?